benchmarks/benchmark-flink/src/main/java/cz/seznam/euphoria/benchmarks/flink/Util.java

```diff
@@ -61,9 +60,7 @@ public long extractTimestamp(Tuple2<Long, T> element) {
   static DataSet<Tuple2<Long, String>> getHdfsSource(ExecutionEnvironment env, URI inputPath)
   throws IOException {
     SearchEventsParser parser = new SearchEventsParser();
-    return env.readHadoopFile(new TextInputFormat(), LongWritable.class, Text.class, inputPath.toString())
-            .map(t -> t.f1.toString())
-            .returns(TypeInformation.of(String.class))
```

Comment:
Don't really see why would this do that. The chain of operations is flatmap-based, so there should be (ideally) no serialization and deserialization involved when applying these transforms. Type hint associated with the return value remained intact. We should however somehow automate performance tests, because otherwise we use this simply as a more complex example code.